Mage.Sets/src/mage/cards/a/AbsorbIdentity.java

```diff
@@ -0,0 +1,80 @@
+package mage.cards.a;
+
+import java.util.UUID;
+
+import mage.abilities.Ability;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.ReturnToHandTargetEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.constants.SubType;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.common.TargetCreaturePermanent;
+import mage.util.functions.EmptyApplyToPermanent;
+
+/**
+ *
+ * @author weirddan455
+ */
+public final class AbsorbIdentity extends CardImpl {
+
+    public AbsorbIdentity(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.INSTANT}, "{1}{U}");
+
+        // Return target creature to its owner's hand.
+        this.getSpellAbility().addTarget(new TargetCreaturePermanent());
+        this.getSpellAbility().addEffect(new ReturnToHandTargetEffect());
+        // You may have Shapeshifters you control become copies of that creature until end of turn.
```

Comment:
1. Use empty line between abilities;
2. Use standard card's code order: ability -> effect -> target -- it allows to read a code more easy;

Reply:
Ok, just pushed a 2nd commit to clean that up.